app/components/candidate_interface/rejection_reasons/reasons_for_rejection_component.rb

```diff
@@ -0,0 +1,29 @@
+##
+# This component class supports the rendering of rejection reasons from the initial iteration of structured rejection reasons.
+# See https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/docs/reasons-for-rejection.md
+#
+module CandidateInterface
+  class RejectionReasons::ReasonsForRejectionComponent < ViewComponent::Base
+    include ViewHelper
+
+    attr_reader :application_choice
+
+    def initialize(application_choice:, render_link_to_find_when_rejected_on_qualifications: false)
+      @application_choice = application_choice
+      @render_link_to_find_when_rejected_on_qualifications = render_link_to_find_when_rejected_on_qualifications
+    end
+
+    def link_to_find_when_rejected_on_qualifications(application_choice)
```

Comment:
Does this method need an argument when you got `@application_choice`?

Reply:
I don't think it does. :+1: